### PR TITLE
Add Core extension integration

### PR DESCRIPTION
**Pull request by laviniat1996**
This PR adds Core as the main wallet option to our docs. This means that MM was replaced with Core in every every tutorial.

Titles of pages where it was replaced:

- Build your first subnet
- Deploying Subnet with EVM Based Blockchain using AvalancheJS
- Introduction to ERC-721 (NFT) Smart Contracts
- Create an ERC-20 Token Using Solidity
- Deploy a Smart Contract on Avalanche Using Remix and MetaMask
- Adjusting Gas Price During High Network Activity
- Fuji Workflow
- Avalanche Subnet Faucet
- Deploy a Permissioned Subnet on Testnet 
- Deploy a Smart Contract on Your Subnet-EVM Using Remix and MetaMask
- Deploying Cross-Chain EVM <-> EVM Bridge
- Stateful Precompile Generation Tutorial
- WAGMI Subnet Demo

Of course, new screenshots and instructions were added where needed. 

**Minor changes can be found in other files too (ex: some references).**

### CODE REVIEW

File: docs/dapps/smart-contracts/deploy-a-smart-contract-on-avalanche-using-remix-and-core.md [hunk 1]
```diff
@@ -19,29 +19,23 @@ DApps and the user experience.
...
+[guide](https://support.avax.network/en/articles/6100129-core-extension-how-do-i-create-a-new-wallet)
+to create a new wallet. 
 
-![MetaMask network dropdown](</img/image(60).png>)
+If you want to use the **Avalanche C-Chain**, it can be selected from the networks list.
 
-### Avalanche Mainnet Settings
+To switch to the **Fuji test network**, go to Settings, select Advanced, and then turn Testnet Mode on.
 
-- **Network Name**: Avalanche Mainnet C-Chain
```
Comment (meaghanfitzgerald) [inline docs/dapps/smart-contracts/deploy-a-smart-contract-on-avalanche-using-remix-and-core.md:30]: Are these settings preserved in a reference section somewhere? I think it is important we have it somewhere incase a MM user wants to add the network and test network to MM

Comment (laviniat1996) [reply docs/dapps/smart-contracts/deploy-a-smart-contract-on-avalanche-using-remix-and-core.md:30]: Hmm, I think I only kept Fuji in a reference section. But good point.

File: docs/dapps/nfts/intro-to-erc721s.md [hunk 2]
```diff
@@ -44,20 +44,40 @@ Once the image and metadata files are ready, we can prepare to deploy a smart co
...
+![Settings image 1](/img/c-chain-ERC20/settings1.png)
+
+Here, turn on the **Testnet Mode** feature. This will automatically make Core switch to
+Fuji Testnet. 
+
+![Settings image 2](/img/c-chain-ERC20/settings2.png)
+
+:::info
+
+If you are using other wallets, like Core or MetaMask, you can add the Fuji
```
Comment (amandamarquis1) [inline docs/dapps/nfts/intro-to-erc721s.md:63]: I would reword this slightly - the sentence above mentions how to toggle on the Tesnet mode in Core, however, this line could be confusing saying "wallets, like **Core** or MetaMask...".

Comment (laviniat1996) [reply docs/dapps/nfts/intro-to-erc721s.md:63]: That's true, thank you. I will remove Core from there.

**Comment by meaghanfitzgerald** [comment]
I can work with you to make a dedicated page for the network configs necessary to add C-chain and others to Metamask

**Comment by meaghanfitzgerald** [comment]
going to push these changes today and plan for network configs in the docs revamp

**Review by meaghanfitzgerald — APPROVED** [review]
